AppController/djinn.rb

```diff
@@ -2125,11 +2145,9 @@ def job_start(secret)
       check_stopped_apps()
 
       # Login nodes may need to check/update nginx/haproxy.
-      if my_node.is_load_balancer?
-        APPS_LOCK.synchronize {
-          regenerate_routing_config()
-        }
-      end
+      APPS_LOCK.synchronize {
```

Comment:
Is there a reason to block at this point if something else has the lock and this node is not the load balancer?